src/smart-components/request/request-detail/request.js

```diff
@@ -123,7 +128,7 @@ class Request extends Component {
           <Stack gutter="md">
             <StackItem>
               <TextContent component={ TextVariants.h6 }>
-                { this.fetchRequestDetails(item) }
+                { fetchRequestDetails(item) }
```

Comment:
We should probably use different name for this function. It looks like we are doing some async call while rendering.